modules/mobile/app/services/mobile/v0/appointments/proxy.rb

```diff
@@ -6,152 +6,17 @@ module Mobile
   module V0
     module Appointments
       class Proxy
```

Comment:
Should these couple of functions remain in the Proxy? Is seems to no longer serve my understanding of a Proxy, but I wasn't sure

Reply:
my vote is to move them to the facility helper.

Reply:
Moved it into the helper. lmk if I should change implementation